src/web/mjs/connectors/MangaRaw.mjs

```diff
@@ -0,0 +1,78 @@
+import Connector from '../engine/Connector.mjs';
+import Manga from '../engine/Manga.mjs';
+
+export default class MangaRaw extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'mangaraw';
+        super.label = 'MangaRaw (Manga Raw Club)';
+        this.tags = ['multi-lingual', 'manga'];
+        this.url = 'https://www.manga-raw.club';
+    }
+
+    async _getMangas() {
+        let mangaList = [];
+        let request = new Request(this.url +'/browse/', this.requestOptions);
+        let data = await this.fetchDOM(request, '#explore > nav.paging a:nth-last-of-type(2)');
+        let pageCount = parseInt(data[0].href.match(/results=(\d+)&/)[1]);
+        for(let page = 1; page <= pageCount; page++) {
+            let mangas = await this._getMangasFromPage(page);
+            mangaList.push(...mangas);
+        }
+        return mangaList;
+    }
+
+    async _getMangasFromPage(page) {
+        let request = new Request(`${this.url}/browse/?results=${page}`, this.requestOptions);
+        let data = await this.fetchDOM(request, '.novel-item a',3);
+        await this.wait(250);
+        return data.map(element => {
+            return {
+                id: this.getRootRelativeOrAbsoluteLink(element, this.url),
+                title: element.title.trim()
+            };
+        });
+    }
+
+    async _getMangaFromURI(uri) {
+        let request = new Request(uri, this.requestOptions);
+        let data = await this.fetchDOM(request, '#novel');
+        let id = uri.pathname + uri.search;
+        let title = data[0].dataset.novelstr.trim();
+        return new Manga(this, id, title);
+    }
+
+    async _getPages(chapter) {
+        let request = new Request(new URL(chapter.id, this.url), this.requestOptions);
+        let data = await this.fetchDOM(request, 'source.loading');
+        return data.map(element => this.getAbsolutePath(element.src, request.url));
+    }
+
+    async _getChapters(manga) {
+        let script = `
+        new Promise(resolve => {
+            let chaplist = [];
+            showDivEn();
+            [...document.querySelectorAll('ul#raw > li > a')].map(data => {
```

Comment:
Either assign chapterlist directly with `map` or use `forEach` instead of `map`.
The `map` function should not be used as a replacement for a loop.
`chapters = [...document.querySelectorAll(...)].map(element => ...);